backend/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I suspect that this files also does not belong in the repository.

Reply:
`yarn.lock` should be kept in SC, see https://github.com/yarnpkg/yarn/issues/1583#issuecomment-257544217